File: app/src/test/java/com/papi/nova/shadows/ShadowSpaceParticleView.java
package com.papi.nova.shadows;

import com.papi.nova.ui.SpaceParticleView;

import org.robolectric.annotation.Implements;
import org.robolectric.shadows.ShadowView;

/**
 * Shadow for SpaceParticleView — extends ShadowView so Robolectric
 * can inflate it without the Canvas/particle system initialization.
 */
@Implements(SpaceParticleView.class)
public class ShadowSpaceParticleView extends ShadowView {
}

File: app/src/test/java/com/papi/nova/shadows/ShadowSpaceParticleView.kt
package com.papi.nova.shadows

import com.papi.nova.ui.SpaceParticleView
import org.robolectric.annotation.Implements
import org.robolectric.shadows.ShadowView

@Implements(SpaceParticleView::class)
class ShadowSpaceParticleView : ShadowView()
